sklearn/cluster/_agglomerative.py

```diff
@@ -750,7 +753,7 @@ class AgglomerativeClustering(ClusterMixin, BaseEstimator):
 
     affinity : str or callable, default='euclidean'
         Metric used to compute the linkage. Can be "euclidean", "l1", "l2",
-        "manhattan", "cosine", or "precomputed".
+        "manhattan", "cosine", "cityblock" or "precomputed".
```

Comment:
I assume that we could make a similar documentation to `DBSCAN` (i.e. `metric` parameter) to state the valid option.

Reply:
The PR for parameter validation in DBSCAN (https://github.com/scikit-learn/scikit-learn/pull/23704) has already been approved. So I guess, this needs to be done in a separate PR then.